pkg/server/operands_handler.go

```diff
@@ -92,3 +94,63 @@ func (o *OperandsListHandler) OperandsListHandler(user *auth.User, w http.Respon
 	}
 	w.Write(resp)
 }
+
+func (o *OperandsListHandler) OperatorListHandler(user *auth.User, w http.ResponseWriter, r *http.Request) {
```

Comment:
I think I mentioned this, but it's really confusing that we have a handler named "OperatorListHandler", which is a member function of an "OperandsListHandler" type, and returns a "PackageManifest" resource. I know this is a WIP so naming/API is probably not finalized, but wanted to point it out early.

Reply:
👍